apps/api/src/billing/lib/batch-signing-client/batch-signing-client.service.ts

```diff
@@ -63,6 +64,7 @@ export class BatchSigningClientService {
     private readonly wallet: Wallet,
     private readonly registry: Registry,
     private readonly connectWithSigner: ConnectWithSignerFn,
+    private readonly chainNetworkConfigService: ChainNetworkConfigService,
```

Comment:
the idea behind modular design and config per module is that every module declares what it needs. E.g. if several modules need rpc url both would declare it in their config. This imo allows to loose coupling between modules and make it more transparent what every module depends on.